website/static/css/commentpane.css

```diff
@@ -1,42 +1,46 @@
-#commentPane {
+.commentPane {
     position: fixed;
-    z-index: 98; /* Note: Must be lower than Bootstrap modals, BlockUI */
+    z-index: 999; /* Note: Must be lower than Bootstrap modals, BlockUI */
```

Comment:
It actually does (when the comment pane is open I cannot click on "Settings" or "Discussion" (the new tab in this PR). But I think it would be a bigger problem if you do not see the comment pane trigger at all